compiler/plugins/input/Torch/InputConversion/ConvertTMTensorToLinalgExt.cpp

```diff
@@ -102,8 +102,24 @@ struct AttentionOpConversion
     Value value = op.getValue();
 
     ShapedType outputType = op.getOutputType();
+
+    SmallVector<int64_t> staticSizes;
+    SmallVector<Value> dynSizes;
+    for (int i = 0, s = outputType.getRank() - 1; i < s; ++i) {
+      staticSizes.push_back(outputType.getDimSize(i));
+      if (outputType.isDynamicDim(i)) {
+        dynSizes.push_back(rewriter.create<tensor::DimOp>(loc, query, i));
+      }
+    }
+
+    staticSizes.push_back(outputType.getDimSize(outputType.getRank() - 1));
+    if (staticSizes.back() == ShapedType::kDynamic) {
+      dynSizes.push_back(
+          rewriter.create<tensor::DimOp>(loc, value, outputType.getRank() - 1));
+    }
+
     Value result = rewriter.create<tensor::EmptyOp>(
-        loc, outputType.getShape(), outputType.getElementType());
```

Comment:
Can we just use the old outputType.getShape() instead of we reconstructiong staticSizes?

Reply:
.... yes. Fixed.